### PR TITLE
New state eligibility dates

### PR DESCRIPTION
**Pull request by harcod**
New vaccination eligibility dates.

image

### CODE REVIEW

File: src/components/StateEligibility.js [hunk 1]
```diff
@@ -100,16 +100,53 @@ const criteriaGroups = [
             },
         ],
     },
-    /* TODO - remove the following div after March 11, and update link to be appropriate link from https://www.mass.gov/covid-19-vaccine */
+    // -------------------------------------------------------------------------
     {
-        endDate: "2021-03-11T00:00:00-05:00", // current timezone offset is at the end
-        title: "Eligible to sign up starting March 11: ",
+        endDate: "2021-04-19T00:00:00-04:00", // current timezone offset is at the end
```
Comment (tkjones117) [inline src/components/StateEligibility.js:105]: Why will this have an `endDate` of 4/19? Maybe I'm forgetting what `endDate` means. But shouldn't in be 3/22? Same for the April 5 `endDate` below.

Comment (harcod) [reply src/components/StateEligibility.js:105]: This is a fast fix.  I will address all of the end dates in another issue

Comment (tkjones117) [reply src/components/StateEligibility.js:105]: ok. maybe you can remove the `3-11` startDate above then too

File: src/components/StateEligibility.js [hunk 2]
```diff
@@ -100,16 +100,53 @@ const criteriaGroups = [
...
+        ],
+    },
+    // -------------------------------------------------------------------------
+    {
+        endDate: "2021-04-19T00:00:00-04:00", // current timezone offset is at the end
+        title: "Eligible to sign up starting April 19: ",
+        list: [
+            {
+                link:
+                    "https://www.mass.gov/info-details/massachusetts-covid-19-vaccination-phases#phase-3-",
```
Comment (tkjones117) [inline src/components/StateEligibility.js:148]: This link doesn't say anything about age 16 and older?

Comment (harcod) [reply src/components/StateEligibility.js:148]: I just linked to the same place that the State's site linked to.

Comment (harcod) [reply src/components/StateEligibility.js:148]: See https://www.mass.gov/covid-19-vaccine for the reference

**Review by tkjones117 — APPROVED** [review]
Code changes lgtm, and the I ran the code locally. Way to get this out fast ⚡

**Review by livgust — APPROVED** [review]

**Pull request merged**